test/org/centos/contra/Infra/Utils.groovy

```diff
@@ -0,0 +1,347 @@
+import spock.lang.*
```

Comment:
You shouldn't need to do this import

Reply:
That import is where `@Shared` comes from

Reply:
So, my next question would be, why are we sharing those variables through all the tests? Each test should be completely separate and start from scratch each time. So if you are sharing `infraUtils` and make changes to something in the first test, the second test is now starting with a modified Utils instead of one that is new.

Reply:
Nothing that's being shared is being modified inside the tests. It's loading the resources that all the tests need, so the idea is to reduce overhead by only loading them once.

Reply:
I guess i don't it being that expensive to load multiple times. And in my opinion we want to isolate our tests to avoid an unintended consequences of sharing resources between the tests